// lib0/dist/buffer.cjs
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

require('./string-e7eb0c1d.cjs');
require('./environment-01f6fae6.cjs');
var encoding = require('./buffer-1008178b.cjs');
require('./array-fd99bc23.cjs');
require('./set-0f209abb.cjs');
require('./map-9a5915e4.cjs');
require('./conditions-f5c0c102.cjs');
require('./storage.cjs');
require('./function-b07b935b.cjs');
require('./object-aad630ed.cjs');
require('./binary-ac8e39e2.cjs');
require('./math-08e068f9.cjs');
require('./number-dc113762.cjs');
require('./error-8582d695.cjs');



exports.copyUint8Array = encoding.copyUint8Array;
exports.createUint8ArrayFromArrayBuffer = encoding.createUint8ArrayFromArrayBuffer;
exports.createUint8ArrayFromLen = encoding.createUint8ArrayFromLen;
exports.createUint8ArrayViewFromArrayBuffer = encoding.createUint8ArrayViewFromArrayBuffer;
exports.decodeAny = encoding.decodeAny;
exports.encodeAny = encoding.encodeAny;
exports.fromBase64 = encoding.fromBase64;
exports.toBase64 = encoding.toBase64;
