// lib0/dist/decoding.cjs
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

var encoding = require('./buffer-1008178b.cjs');
require('./binary-ac8e39e2.cjs');
require('./math-08e068f9.cjs');
require('./number-dc113762.cjs');
require('./string-e7eb0c1d.cjs');
require('./error-8582d695.cjs');
require('./environment-01f6fae6.cjs');
require('./map-9a5915e4.cjs');
require('./conditions-f5c0c102.cjs');
require('./storage.cjs');
require('./function-b07b935b.cjs');
require('./array-fd99bc23.cjs');
require('./set-0f209abb.cjs');
require('./object-aad630ed.cjs');



exports.Decoder = encoding.Decoder;
exports.IncUintOptRleDecoder = encoding.IncUintOptRleDecoder;
exports.IntDiffDecoder = encoding.IntDiffDecoder;
exports.IntDiffOptRleDecoder = encoding.IntDiffOptRleDecoder;
exports.RleDecoder = encoding.RleDecoder;
exports.RleIntDiffDecoder = encoding.RleIntDiffDecoder;
exports.StringDecoder = encoding.StringDecoder;
exports.UintOptRleDecoder = encoding.UintOptRleDecoder;
exports._readVarStringNative = encoding._readVarStringNative;
exports._readVarStringPolyfill = encoding._readVarStringPolyfill;
exports.clone = encoding.clone;
exports.createDecoder = encoding.createDecoder;
exports.hasContent = encoding.hasContent$1;
exports.peekUint16 = encoding.peekUint16;
exports.peekUint32 = encoding.peekUint32;
exports.peekUint8 = encoding.peekUint8;
exports.peekVarInt = encoding.peekVarInt;
exports.peekVarString = encoding.peekVarString;
exports.peekVarUint = encoding.peekVarUint;
exports.readAny = encoding.readAny;
exports.readBigInt64 = encoding.readBigInt64;
exports.readBigUint64 = encoding.readBigUint64;
exports.readFloat32 = encoding.readFloat32;
exports.readFloat64 = encoding.readFloat64;
exports.readFromDataView = encoding.readFromDataView;
exports.readTailAsUint8Array = encoding.readTailAsUint8Array;
exports.readUint16 = encoding.readUint16;
exports.readUint32 = encoding.readUint32;
exports.readUint32BigEndian = encoding.readUint32BigEndian;
exports.readUint8 = encoding.readUint8;
exports.readUint8Array = encoding.readUint8Array;
exports.readVarInt = encoding.readVarInt;
exports.readVarString = encoding.readVarString;
exports.readVarUint = encoding.readVarUint;
exports.readVarUint8Array = encoding.readVarUint8Array;
exports.skip8 = encoding.skip8;
