// lib0/dist/encoding.cjs
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

var encoding = require('./buffer-1008178b.cjs');
require('./math-08e068f9.cjs');
require('./number-dc113762.cjs');
require('./binary-ac8e39e2.cjs');
require('./string-e7eb0c1d.cjs');
require('./array-fd99bc23.cjs');
require('./environment-01f6fae6.cjs');
require('./map-9a5915e4.cjs');
require('./conditions-f5c0c102.cjs');
require('./storage.cjs');
require('./function-b07b935b.cjs');
require('./object-aad630ed.cjs');
require('./error-8582d695.cjs');
require('./set-0f209abb.cjs');



exports.Encoder = encoding.Encoder;
exports.IncUintOptRleEncoder = encoding.IncUintOptRleEncoder;
exports.IntDiffEncoder = encoding.IntDiffEncoder;
exports.IntDiffOptRleEncoder = encoding.IntDiffOptRleEncoder;
exports.RleEncoder = encoding.RleEncoder;
exports.RleIntDiffEncoder = encoding.RleIntDiffEncoder;
exports.StringEncoder = encoding.StringEncoder;
exports.UintOptRleEncoder = encoding.UintOptRleEncoder;
exports._writeVarStringNative = encoding._writeVarStringNative;
exports._writeVarStringPolyfill = encoding._writeVarStringPolyfill;
exports.createEncoder = encoding.createEncoder;
exports.hasContent = encoding.hasContent;
exports.length = encoding.length;
exports.set = encoding.set;
exports.setUint16 = encoding.setUint16;
exports.setUint32 = encoding.setUint32;
exports.setUint8 = encoding.setUint8;
exports.toUint8Array = encoding.toUint8Array;
exports.verifyLen = encoding.verifyLen;
exports.write = encoding.write;
exports.writeAny = encoding.writeAny;
exports.writeBigInt64 = encoding.writeBigInt64;
exports.writeBigUint64 = encoding.writeBigUint64;
exports.writeBinaryEncoder = encoding.writeBinaryEncoder;
exports.writeFloat32 = encoding.writeFloat32;
exports.writeFloat64 = encoding.writeFloat64;
exports.writeOnDataView = encoding.writeOnDataView;
exports.writeUint16 = encoding.writeUint16;
exports.writeUint32 = encoding.writeUint32;
exports.writeUint32BigEndian = encoding.writeUint32BigEndian;
exports.writeUint8 = encoding.writeUint8;
exports.writeUint8Array = encoding.writeUint8Array;
exports.writeVarInt = encoding.writeVarInt;
exports.writeVarString = encoding.writeVarString;
exports.writeVarUint = encoding.writeVarUint;
exports.writeVarUint8Array = encoding.writeVarUint8Array;
